import sys
import os
myDir = os.getcwd()
sys.path.append(myDir)

from Controllers.SalidaController import marca_primera, marca_ultima, id_primero, id_ultimo, precio_primero, precio_ultimo

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_PaginaSalida(object):

    def setupUi(self, PaginaSalida):
        PaginaSalida.setObjectName("PaginaSalida")
        PaginaSalida.resize(800, 600)
        PaginaSalida.setStyleSheet("background-color: rgb(51, 51, 51);")
        self.centralwidget = QtWidgets.QWidget(PaginaSalida)
        self.centralwidget.setObjectName("centralwidget")
        self.btn_Menu = QtWidgets.QPushButton(self.centralwidget)
        self.btn_Menu.setGeometry(QtCore.QRect(20, 20, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setBold(True)
        font.setWeight(75)
        self.btn_Menu.setFont(font)
        self.btn_Menu.setStyleSheet("border-style: solid;\n"
"border-width: 3px;\n"
"background-color: rgb(236, 236, 236);\n"
"color: rgb(51, 51, 51);\n"
"border-radius: 10px;")
        self.btn_Menu.setObjectName("btn_Menu")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(140, 60, 601, 81))
        font = QtGui.QFont()
        font.setFamily("Microsoft JhengHei UI")
        font.setPointSize(28)
        self.label_3.setFont(font)
        self.label_3.setStyleSheet("background-color: rgb(51, 51, 51);\n"
"color: rgb(186, 0, 0);")
        self.label_3.setObjectName("label_3")
        self.btn_busqueda = QtWidgets.QPushButton(self.centralwidget)
        self.btn_busqueda.setGeometry(QtCore.QRect(150, 180, 231, 51))
        font = QtGui.QFont()
        font.setPointSize(16)
        self.btn_busqueda.setFont(font)
        self.btn_busqueda.setStyleSheet("border-style: solid;\n"
"border-width: 3px;\n"
"border-color: rgb(186, 0, 0);\n"
"background-color: rgb(51, 51, 51);\n"
"color: rgb(186, 0, 0);\n"
"border-radius: 25px;")
        self.btn_busqueda.setObjectName("btn_busqueda")
        self.btn_refrescar = QtWidgets.QPushButton(self.centralwidget)
        self.btn_refrescar.setGeometry(QtCore.QRect(500, 180, 211, 51))
        font = QtGui.QFont()
        font.setPointSize(16)
        self.btn_refrescar.setFont(font)
        self.btn_refrescar.setStyleSheet("border-style: solid;\n"
"border-width: 3px;\n"
"border-color: rgb(186, 0, 0);\n"
"background-color: rgb(51, 51, 51);\n"
"color: rgb(186, 0, 0);\n"
"border-radius: 25px;")
        self.btn_refrescar.setObjectName("btn_refrescar")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(260, 290, 191, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setStyleSheet("color: rgb(236, 236, 236);")
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(530, 290, 191, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_2.setObjectName("label_2")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(110, 350, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.label_4.setFont(font)
        self.label_4.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(110, 430, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.label_5.setFont(font)
        self.label_5.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(110, 510, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.label_6.setFont(font)
        self.label_6.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.centralwidget)
        self.label_7.setGeometry(QtCore.QRect(340, 350, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_7.setFont(font)
        self.label_7.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.centralwidget)
        self.label_8.setGeometry(QtCore.QRect(340, 510, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_8.setFont(font)
        self.label_8.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_8.setObjectName("label_8")
        self.label_9 = QtWidgets.QLabel(self.centralwidget)
        self.label_9.setGeometry(QtCore.QRect(340, 430, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_9.setFont(font)
        self.label_9.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_9.setObjectName("label_9")
        self.label_10 = QtWidgets.QLabel(self.centralwidget)
        self.label_10.setGeometry(QtCore.QRect(600, 430, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_10.setFont(font)
        self.label_10.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_10.setObjectName("label_10")
        self.label_11 = QtWidgets.QLabel(self.centralwidget)
        self.label_11.setGeometry(QtCore.QRect(600, 510, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_11.setFont(font)
        self.label_11.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_11.setObjectName("label_11")
        self.label_12 = QtWidgets.QLabel(self.centralwidget)
        self.label_12.setGeometry(QtCore.QRect(600, 350, 81, 31))
        font = QtGui.QFont()
        font.setFamily("Nirmala UI Semilight")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_12.setFont(font)
        self.label_12.setStyleSheet("color: rgb(236, 236, 236);")
        self.label_12.setObjectName("label_12")
        PaginaSalida.setCentralWidget(self.centralwidget)

        #Eventos

        
        #Fin
        
        self.retranslateUi(PaginaSalida)
        QtCore.QMetaObject.connectSlotsByName(PaginaSalida)

    def retranslateUi(self, PaginaSalida):
        _translate = QtCore.QCoreApplication.translate
        PaginaSalida.setWindowTitle(_translate("PaginaSalida", "MainWindow"))
        self.btn_Menu.setText(_translate("PaginaSalida", "Menu"))
        self.label_3.setText(_translate("PaginaSalida", "Programaci√≥n de salidas de autos"))
        self.btn_busqueda.setText(_translate("PaginaSalida", "Realizar busqueda"))
        self.btn_refrescar.setText(_translate("PaginaSalida", "Refrescar"))
        self.label.setText(_translate("PaginaSalida", "Auto proximo a salir"))
        self.label_2.setText(_translate("PaginaSalida", "Auto ultimo en salir"))
        self.label_4.setText(_translate("PaginaSalida", "ID_Auto"))
        self.label_5.setText(_translate("PaginaSalida", "Marca"))
        self.label_6.setText(_translate("PaginaSalida", "Precio"))
        self.label_7.setText(_translate("PaginaSalida", id_primero))
        self.label_8.setText(_translate("PaginaSalida", precio_primero ))
        self.label_9.setText(_translate("PaginaSalida", marca_primera ))
        self.label_10.setText(_translate("PaginaSalida", marca_ultima))
        self.label_11.setText(_translate("PaginaSalida", precio_ultimo))
        self.label_12.setText(_translate("PaginaSalida", id_ultimo))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    PaginaSalida = QtWidgets.QMainWindow()
    ui = Ui_PaginaSalida()
    ui.setupUi(PaginaSalida)
    PaginaSalida.show()
    sys.exit(app.exec_())
